make-rules/ips.mk

```diff
@@ -484,6 +484,10 @@ $(BUILD_DIR) $(MANGLED_DIR):
 	$(MKDIR) $@
 
 PKGMANGLE_OPTIONS = -D $(MANGLED_DIR) $(PKG_PROTO_DIRS:%=-d %)
+ifeq ($(strip $(USE_CTF)),yes)
+CTFCONVERT=/opt/onbld/bin/$(MACH)/ctfconvert
```

Comment:
I think this could be usable even without `USE_CTF` so I suggest to move this to `shared-macros.mk` and define it there unconditionally.

Reply:
Done.